app/components/wallet/ada-redemption/AdaRedemptionForm.js

```diff
@@ -137,8 +141,12 @@ export default class AdaRedemptionForm extends Component {
         value: '',
         bindings: 'ReactToolbox',
         validate: ({ field }) => {
-          const isValid = field.value.length === 44;
-          return [isValid, this.context.intl.formatMessage(messages.redemptionCodeError)];
+          const value = this.props.redemptionCode ? this.props.redemptionCode : field.value;
```

Comment:
@DominikGuzei: when you have time please take a look at this line. When we are populating redemption code from the parsed certificate `field.value` here is always `''` even though field is correctly populated.